src/components/Home.css

```diff
@@ -1,3 +1,304 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;600;700&family=Roboto+Slab:wght@100;300;400;700&display=swap');
+
 .booklist {
-  display: block;
+  margin: 2.313rem 2rem;
+}
+
+.listing {
+  width: 100%;
+  background-color: #fff;
+  display: flex;
+  justify-content: space-between;
+  padding: 30px;
+  border: 1px solid #e8e8e8;
+}
+
+.booklist ul {
+  flex-direction: column;
+  gap: 0.938rem;
+  padding-bottom: 2.5rem;
+}
```

Comment:
- [ ] Great job on the overall styling! But right now your website has horizontal scroll as you can see in the image below. This usually happens when the size of one of the elements on your page is greater than the total screen size which causes the whole website to scroll to be able to fit the content. RIght now that seems like the form. This can be avoided if you use percentages instead of fixed values. Kindly alter this to stop the horizontal scroll as it is bad UX.

![image](https://user-images.githubusercontent.com/90556221/185601339-9d7f55c6-322b-4876-87b8-d727a5144735.png)

- [ ] Additionally, your book cards are too big horizontally and do not conform to the Zeppelin design. Make sure there is lesser padding on the right of each book card as I've highlighted above.

Reply:
@zhadier I have removed horizontal scroll as requested by applying relevant css rules